### PR TITLE
gdal_contour: add page

### PR DESCRIPTION
**Pull request by JakobMiksch**
Add the CLI tool [gdal_contour](https://gdal.org/programs/gdal_contour.html) from the GDAL/OGR suite.

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
3.2.2

### CODE REVIEW

File: pages/common/gdal_contour.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# gdal_contour
+
+> Create contour lines and polygons from a digital elevation model.
+> More information: <https://gdal.org/programs/gdal_contour.html>.
+
+- Create a vector dataset with contour lines every 100 meter and name the elevation property "ele":
+
+`gdal_contour -a {{ele}} -i {{100.0}} {{path/to/input.tif}} {{path/to/output.gpkg}}`
+
+- Create a vector dataset with polygons every 100 meter:
```
Comment (reinhart1010) [inline pages/common/gdal_contour.md:10]:
```suggestion
- Create a vector dataset with [p]olygons spread over an 100-meter [i]nterval:
```

File: pages/common/gdal_contour.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# gdal_contour
+
+> Create contour lines and polygons from a digital elevation model.
+> More information: <https://gdal.org/programs/gdal_contour.html>.
+
+- Create a vector dataset with contour lines every 100 meter and name the elevation property "ele":
```
Comment (reinhart1010) [inline pages/common/gdal_contour.md:6]:
```suggestion
- Create a vector dataset with contour lines spread over a 100-meter [i]nterval while [a]ttributing the elevation property as "ele":
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
It might be better to place some mnemonic tags here, as `gdal_contour` unfortunately doesn't put longer parameter names (e.g. `--attribute` instead of `-a`) which are more preferred for writing tldr-pages.

**Comment by JakobMiksch** [comment]
@reinhart1010 thanks for your review, I applied your suggestions.

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**